logging/log.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 Fraunhofer AISEC
```

Comment:
a nice idea, but can we do that in a separate PR? Or is it needed for the tests?